dependencies/graknlabs/dependencies.bzl

```diff
@@ -20,8 +20,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def graknlabs_dependencies():
     git_repository(
         name = "graknlabs_dependencies",
-        remote = "https://github.com/graknlabs/dependencies",
-        commit = "c921c9468468249764eebafa9940a487f577c3f8",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_dependencies
+        remote = "https://github.com/vmax/graknlabs-dependencies",
+        commit = "ec9462cc70ce7366385ac12f83d20906523acd96",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_dependencies
```

Comment:
Is there a PR on `@graknlabs_dependencies` for this already? That should be merged first before this PR goes in.

Reply:
@haikalpribadi yes (graknlabs/dependencies#157), we'll make sure it gets merged before this one